graphs/possible_bipartition.py

```diff
@@ -1,12 +1,40 @@
 # Can be used for BFS
 from collections import deque 
 
+color_list = ["blue, orange"]
```

Comment:
To assign the dogs into two different colored groups, your color list needs to have multiple items. 

This bug appears in the instructor solution as well. 
```suggestion
color_list = ["blue", "orange"]
```